components/scripts/lib/cli-parsers/common.m4

```diff
@@ -3,6 +3,7 @@
 # ARG_OPTIONAL_BOOLEAN([enable-gradle-enterprise],[e],[])
 # ARG_OPTIONAL_SINGLE([git-branch],[b],[])
 # ARG_OPTIONAL_SINGLE([git-commit-id],[c],[])
+# ARG_OPTIONAL_SINGLE([git-options],[o],[])
```

Comment:
In interactive mode, we need to print the `-o` value in the end-of-experiment summary if it differs from the default. Currently, we don't.

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/231070/197924474-60c2e991-2571-4992-8661-9ff5c9cc6fb5.png">